bundles/ranvier-commands/commands/get.js

```diff
@@ -63,6 +63,7 @@ module.exports = (srcPath) => {
       player.addItem(item);
 
       Broadcast.sayAt(player, `<green>You receive loot: </green>${item.display}<green>.</green>`);
+      Broadcast.sayAt(room, `<green>${player.name} receives loot: </green>${item.display}<green>.</green>`)
```

Comment:
I think eventually there could be a `spam` command to allow you to disable that type of output but until then I think it's best to not have it